redi-connect-backend/lib/email/email.js

```diff
@@ -463,6 +296,88 @@ const sendVerificationEmail = ({ recipient, redUserId, firstName, userType, veri
   }).subscribe()
 }
 
+const sendMentorPendingReviewDeclinedEmailTemplate = fs.readFileSync(path.resolve(__dirname, 'templates', 'validate-email-address-successful.mjml'), 'utf-8')
+const sendMentorPendingReviewDeclinedEmailParsed = mjml2html(sendMentorPendingReviewDeclinedEmailTemplate, {
+  filePath: path.resolve(__dirname, 'templates')
+})
+
+const sendMentorPendingReviewDeclinedEmail = ({
+  recipient,
+  firstName,
+  rediLocation
+}) => {
+  const html = sendMentorPendingReviewDeclinedEmailParsed.html
+    .replace('${firstName}', firstName)
+  sendMjmlEmailFactory({
+    to: recipient,
+    subject: 'Verification has been successful!',
+    html: html,
+    rediLocation
+  }).subscribe()
+}
+
+const sendMenteePendingReviewDeclinedEmailTemplate = fs.readFileSync(path.resolve(__dirname, 'templates', 'validate-email-address-successful.mjml'), 'utf-8')
+const sendMenteePendingReviewDeclinedEmailParsed = mjml2html(sendMenteePendingReviewDeclinedEmailTemplate, {
+  filePath: path.resolve(__dirname, 'templates')
+})
+
+const sendMenteePendingReviewDeclinedEmail = ({
+  recipient,
+  firstName,
+  rediLocation
+}) => {
+  const html = sendMenteePendingReviewDeclinedEmailParsed.html
+    .replace('${firstName}', firstName)
+  sendMjmlEmailFactory({
+    to: recipient,
+    subject: 'Verification has been successful!',
+    html: html,
+    rediLocation
+  }).subscribe()
+}
+
+const sendMentorPendingReviewAcceptedEmailTemplate = fs.readFileSync(path.resolve(__dirname, 'templates', 'welcome-to-redi.mjml'), 'utf-8')
```

Comment:
As discussed I added a second footer because this is the only difference in the template atm. By setting the `rediLocation` it will return the appropiate footer.